README.md

```diff
@@ -20,25 +20,19 @@
 ###  <div align="left">Turns Data and AI algorithms into full web applications in no time.
 ###  How? Taipy GUI with Taipy Core pops out as a 360° platform to build production-ready web applications</div>
```

Comment:
This text is also present below. Delete this one?

Reply:
updated!